src/switch_rtp.c

```diff
@@ -8215,7 +8215,7 @@ static int rtp_common_write(switch_rtp_t *rtp_session,
 			int delta = rtp_session->ts - rtp_session->last_write_ts;
 
 			if (!rtp_session->flags[SWITCH_RTP_FLAG_UDPTL] &&
-				((!rtp_session->flags[SWITCH_RTP_FLAG_RESET] && (abs(delta) > rtp_session->samples_per_interval * 10))
```

Comment:
This condition looked backwards to me. The inverse will successfully trigger a marker bit insertion. I'd love to hear some thoughts on this.

Reply:
From your last comment to the connected issue I understood that SSRC change is enough to resolve the problem you observe. So why would you like to instert m bit?

Reply:
From my testing, both the marker bit and the SSRC change is necessary to fix the issue.

I see now when reading my previous comments that it seems like I discarded the marker bit entirely and solved it with just the SSRC, but no, both of these changes are needed for this case to be solved. Sorry about that!